src/Raven.Studio/wwwroot/App/views/database/documents/editDocument.html

```diff
@@ -161,6 +168,17 @@ <h3 class="margin-none">Properties</h3>
                 </div>
                 <div class="flex-window-head" data-bind="visible: isUploaderActive, template: { name: 'uploader-template' }">
                 </div>
+                <div class="flex-window-head" data-bind="visible: isCountersActive">
+                    <div class="padding padding-xs margin-top margin-top-sm">
+                        <!--<label class="btn btn-sm btn-primary btn-block"><i class="icon-new-counter" data-bind="$root.setCounter(null)">-->
```

Comment:
put TODO as it is hard to find such places w/o TODO

Reply:
done